metashare/api/models.py

```diff
@@ -295,6 +296,18 @@ def save(self, *args, **kwargs):
         if self.tracker.has_changed("url"):
             self.notify_has_url()
 
+        if self.tracker.has_changed("currently_refreshing_changes"):
+            self.notify_refreshing_changes()
+
+    def queue_delete(self):
+        from .jobs import delete_scratch_org_job
+
+        delete_scratch_org_job.delay(self)
+
+    def delete(self, *args, **kwargs):
+        self.notify_deleted()
```

Comment:
This is now triggered by https://github.com/SFDO-Tooling/MetaShare/pull/151/files#diff-e148a690b7578d0eded316df5a9c8939R52 when a scratch-org-create action fails. That's not ideal, from the perspective of the front end at least, because it can sometimes (depending on WS race conditions) result in a "You have successfully deleted this scratch org" message when really the org just wasn't successfully created. Can we make this more specific somehow, tying the WS event more directly to the action it is describing?